Description: use setuptools for the install target so python*:Depends substitutions work
Author: Julien Puydt
Forwarded: Debian-specific
---
 setup.py | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/setup.py b/setup.py
index 5291928..d480b8a 100755
--- a/setup.py
+++ b/setup.py
@@ -164,8 +164,7 @@
            'egg_info', 'easy_install', 'upload', 'install_egg_info',
           }
 
-if len(needs_setuptools.intersection(sys.argv)) > 0:
-    import setuptools
+import setuptools
 
 # This dict is used for passing extra arguments that are setuptools
 # specific to setup
